### PR TITLE
412- add compare scripts to respective directories

### PR DESCRIPTION
**Pull request by rjbdev05**
## Description
- Commit integration test comparison scripts for R2 PGEs

## Affected Issues
- he S3 locations of the comparison scripts for each of the PGEs can be found here: https://github.com/nasa/opera-sds-pge/blob/main/.ci/scripts/cslc_s1/test_int_cslc_s1.sh#L65 https://github.com/nasa/opera-sds-pge/blob/main/.ci/scripts/rtc_s1/test_int_rtc_s1.sh#L66
## Testing
- Rename the downloaded scripts to <pge_name>_compare.py (ex: rtc_s1_compare.py, cslc_s1_compare.py)
- ommit the renamed scripts to the PGE repository within the corresponding integration test directory (ex: .ci/scripts/cslc_s1)

### CODE REVIEW

File: .ci/scripts/cslc_s1/test_int_cslc_s1.sh [hunk 1]
```diff
@@ -184,7 +184,7 @@ if [ $overall_status -eq 0 ]; then
                                 -v "${expected_dir}":/exp:ro \
                                 --entrypoint /home/compass_user/miniconda3/envs/COMPASS/bin/python3 \
                                 ${PGE_IMAGE}:"${PGE_TAG}" \
-                                /working/validate_product.py \
+                                "${SCRIPT_DIR}"/../cslc/cslc_s1_compare.py \
```
Comment (collinss-jpl) [inline .ci/scripts/cslc_s1/test_int_cslc_s1.sh:187]: Since this script is now part of the repository, we do not need to invoke it as part of a `docker run` command.

Basically, lines 181-190 can be replaced with something like the following:

`compare_out=$("${SCRIPT_DIR}"/../cslc_s1/cslc_s1_compare.py --ref-product ${ref_product} --sec-product ${sec_product} -p CSLC 2>&1) || compare_exit_status=$?`

Usage of `$docker_out` should then be replaced with `$compare_out`. 
This also needs to be applied to lines 220-229 below.

**Comment by rjbdev05** [comment]
I fixed and created pull request – did I do it right?

Ray

From: Scott Collins ***@***.***>
Reply-To: nasa/opera-sds-pge ***@***.***>
Date: Tuesday, March 12, 2024 at 10:23 AM
To: nasa/opera-sds-pge ***@***.***>
Cc: "Bambery, Raymond J (US 398F)" ***@***.***>, Author ***@***.***>
Subject: [EXTERNAL] Re: [nasa/opera-sds-pge] 412- add compare scripts to respective directories (PR #414)


@collinss-jpl commented on this pull request.

________________________________

In .ci/scripts/cslc_s1/test_int_cslc_s1.sh<https://urldefense.us/v3/__https:/github.com/nasa/opera-sds-pge/pull/414*discussion_r1521858357__;Iw!!PvBDto6Hs4WbVuu7!OA3E42fm-3yVonOO1But3AmB2GcuHjMY2K66fHqnBAgVaHDUW_RbSbmXfNCE163ELfrSKm7g3OFv4mPdIbn3sXOqjjWt70wOKu3k$>:

> @@ -184,7 +184,7 @@ if [ $overall_status -eq 0 ]; then

                                 -v "${expected_dir}":/exp:ro \

                                 --entrypoint /home/compass_user/miniconda3/envs/COMPASS/bin/python3 \

                                 ${PGE_IMAGE}:"${PGE_TAG}" \

-                                /working/validate_product.py \

+                                "${SCRIPT_DIR}"/../cslc/cslc_s1_compare.py \

Since this script is now part of the repository, we do not need to invoke it as part of a docker run command.

Basically, lines 181-190 can be replaced with something like the following:

compare_out=$("${SCRIPT_DIR}"/../cslc_s1/cslc_s1_compare.py --ref-product ${ref_product} --sec-product ${sec_product} -p CSLC 2>&1) || compare_exit_status=$?

Usage of $docker_out should then be replaced with $compare_out.
This also needs to be applied to lines 220-229 below.

**Review by collinss-jpl — APPROVED** [review]
